### ISSUE TITLE
Tidy up out put of make_video

### ISSUE THREAD

**Issue by jdpye** [ocean-tracking-network/glatos#57]
In GitLab by @anunes on Feb 21, 2019, 16:10

The output of make_video should just output the file path of the video and not the big ugly string it does now.

```r
ffmpeg version N-93214-g7e4d3dbe18 Copyright (c) 2000-2019 the FFmpeg developers
  built with Apple LLVM version 10.0.0 (clang-1000.11.45.5)
  configuration: --enable-gpl --enable-version3 --enable-sdl2 --enable-fontconfig --enable-gnutls --enable-iconv --enable-libass --enable-libdav1d --enable-libbluray --enable-libfreetype --enable-libmp3lame --enable-libopencore-amrnb --enable-libopencore-amrwb --enable-libopenjpeg --enable-libopus --enable-libshine --enable-libsnappy --enable-libsoxr --enable-libtheora --enable-libtwolame --enable-libvpx --enable-libwavpack --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxml2 --enable-libzimg --enable-lzma --enable-zlib --enable-gmp --enable-libvidstab --enable-libvorbis --enable-libvo-amrwbenc --enable-libmysofa --enable-libspeex --enable-libxvid --enable-libaom --enable-appkit --enable-avfoundation --enable-coreimage --enable-audiotoolbox
  libavutil      56. 26.100 / 56. 26.100
  libavcodec     58. 47.102 / 58. 47.102
  libavformat    58. 26.101 / 58. 26.101
  libavdevice    58.  6.101 / 58.  6.101
  libavfilter     7. 48.100 /  7. 48.100
  libswscale      5.  4.100 /  5.  4.100
  libswresample   3.  4.100 /  3.  4.100
  libpostproc    55.  4.100 / 55.  4.100
Input #0, image2, from '/Users/alexnunes/anaconda3/envs/glatos/lib/R/library/glatos/extdata/frames/%02d.png':
  Duration: 00:00:01.00, start: 0.000000, bitrate: N/A
    Stream #0:0: Video: png, rgb24(pc), 2000x1234, 30 fps, 30 tbr, 30 tbn, 30 tbc
File '/Users/alexnunes/animation.mp4' already exists. Exiting.
[1] "/Users/alexnunes/anaconda3/envs/glatos/lib/R/library/glatos/bin/ffmpeg -framerate 30 -start_number 1 -i \"/Users/alexnunes/anaconda3/envs/glatos/lib/R/library/glatos/extdata/frames/%02d.png\" -c:v libx264 -preset medium -r 30 -vf \"format=yuv420p\" \"/Users/alexnunes/animation.mp4\" -n"
```

Desired output:

```r
Your video has been made in:/Path/to/video.mp4"
```

**Comment by jdpye** [comment]
In GitLab by @anunes on Feb 21, 2019, 16:11

changed the description

**Comment by jdpye** [comment]
In GitLab by @chrisholbrook on Nov 12, 2019, 14:32

mentioned in commit 526d864beb9ac93e0914703955fba2335a7c680a

**Comment by jdpye** [comment]
In GitLab by @chrisholbrook on Nov 12, 2019, 14:38

closed via merge request !18

**Comment by jdpye** [comment]
In GitLab by @chrisholbrook on Nov 12, 2019, 14:38

closed via commit 526d864beb9ac93e0914703955fba2335a7c680a
